docs/serverless-deployment.md

```diff
@@ -0,0 +1,65 @@
+---
+next: docs/persistence.md
+---
+
+# Serverless Deployment
+
+Serverless computing is a model which aims to abstract server management without concerns for implementing, tweaking, or scaling a server (at least, to the perspective of a user). The following sections will provide examples on how to deploy your application to service that will run on-demand as a Function as a Service(FaaS).
+
+To learn more about other FaaS offerings and the concept of serverless, check out the [comparisons and case studies](https://serverless.com/learn/overview) created by the Serverless Framework.
+
+> note: Deploying to FaaS provider will require you to first [create a GitHub App](#create-the-github-app)
+
+**Contents:**
+
+1. [Create the GitHub App](#create-the-github-app)
+1. [Deploy the app to a FaaS provider](#deploy-the-app)
+    1. [AWS Lambda](#aws-lambda)
+    1. [Google Cloud Function](#google-cloud-function)
+
+## Create the GitHub App
+
+Every deployment will need an [App](https://developer.github.com/apps/). If you have not created a GitHub App, you learn how using the [Deployment section of our docs](/deployment/#create-the-github-app)
+
+## Deploy the app
+
+To deploy an app to any cloud provider, you will need 3 environment variables:
+
+- `APP_ID`: the ID of the app, which you can get from the [app settings page](https://github.com/settings/apps).
+- `WEBHOOK_SECRET`: the **Webhook Secret** that you generated when you created the app.
+- `PRIVATE_KEY_PATH`: the path to a private key file
+
+> These environment variables will need to be pass through to your FaaS solution. Each solution is different, please consult their documentation on how to use variables in the deployed environment. 
+
+Choosing a FaaS provider is mostly dependent on developer preference. Each Probot plugin interacts in a similar fashion, but the plugins implementation is dealing with different requests and responses specific to the provider. If you do not have a preference for a provider, choose the solution you have the most familiarity with.
+
+### AWS Lambda
+
+AWS Lambda is an event-driven, serverless computing platform provided by Amazon as a part of the Amazon Web Services. AWS Lamba additionally manages the computing resources required for the code and adjusts those resources in conjunction with incoming events.
+1. [Install the @probot/serverless-lambda](https://github.com/probot/serverless-lambda#usage) plugin.
+2. Create a `handler.js` file in the route of you probot application
```

Comment:
Looks like a couple of misspellings here. Did you mean to say "in the root of your probot application"?

Reply:
root is indeed what was meant.